pkgs/development/python-modules/wxPython/4.0.nix

```diff
@@ -1,28 +1,14 @@
 { lib
 , stdenv
-, openglSupport ? true
-, libX11
-, pyopengl
 , buildPythonPackage
 , fetchPypi
 , pkgconfig
-, libjpeg
```

Comment:
Per discussion above, I do not recommend merging these changes as python packages need to have their runtime dependencies in `buildInputs`, and the developers of wxPython Phoenix indicate that these are dependencies. If it is demonstrated that the tests succeed despite this omission I would be in favor of the change, but am skeptical as is since no tests are running.